### DISCUSSION TITLE
Targeted disabling of styles

### DISCUSSION THREAD

**Discussion by bulats** [tailwindlabs/tailwindcss#2994] [Help] [answered]
I'm trying to disable the following part of tailwind css as there's a conflict with another library:

```
*, ::before, ::after {
    box-sizing: border-box;
    border-width: 0;
    border-style: solid;
    border-color: #e5e7eb;
}
```

I have tried disabling using ```corePlugins```: ```borderStyle``` and ```borderWidth```, however, this doesn't solve the issue. Disabling ```preflight``` fixes the issue but removes too much relevant styling on top. Is there a way around this without other than via ```preflight```?

**Comment by bulats** [comment, marked as answer]
The solution is to use ```@layer```

```
@layer base {
  *, ::before, ::after {
    box-sizing: border-box;
    border-style: none;
    border-color: #e5e7eb;
  }
}

**Reply by simonswiss** [reply]
Yep, that should work - that way you are essentially passing some "override styles" to the `base` layer 👍
